supabase/migrations/20260808071749_apps_with_preview_global_stats.sql

```
@@ -0,0 +1,5 @@
+ALTER TABLE public.global_stats
+  ADD COLUMN IF NOT EXISTS apps_with_preview bigint NOT NULL DEFAULT 0;
+
+COMMENT ON COLUMN public.global_stats.apps_with_preview
+  IS 'Number of apps with preview QR enabled (allow_preview = true) at snapshot day end.';
```

SQLFluff (4.2.2)

[error] Line is too long (90 > 80).

(LT05)